cv_rect rect_r
cv_rect rect_g
cv_rect rect_b